iFixit/react-commerce
# Add featured subcollections section

This PR closes #82 

## Changelog

- Adds featured subcollections section
  ![image](https://user-images.githubusercontent.com/4640135/136405055-f088cc4a-829b-438c-be48-0f70d1873f22.png)
- Fixes subcategories image alignment
- Adds title to related posts

## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Verify that the featured subcollections section is rendered according to data from Strapi
